src/BenchmarkDotNet/Exporters/Json/SimpleJson.cs

```diff
@@ -1554,11 +1554,29 @@ protected virtual bool TrySerializeUnknownTypes(object input, out object output)
             foreach (KeyValuePair<string, ReflectionUtils.GetDelegate> getter in getters)
             {
                 if (getter.Value != null)
-                    obj.Add(MapClrMemberNameToJsonFieldName(getter.Key), getter.Value(input));
+                    obj.Add(MapClrMemberNameToJsonFieldName(getter.Key), ReplaceUnsupportedNumericValues(getter.Value(input)));
             }
             output = obj;
             return true;
         }
+
+        private object ReplaceUnsupportedNumericValues(object value)
+        {
+            if (!(value is double))
+            {
+                return value;
+            }
+
+            switch (value)
+            {
+                case double.NaN:
+                case double.NegativeInfinity:
+                case double.PositiveInfinity:
+                    return 0.0;
```

Comment:
I wonder why we do get `NaN`s here. Most probably by doing division by `0` using doubles?

@marcnet80  In such a case, would it be better (and possible) to return an empty string instead of `0.0`? Just to say "there is no value" instead "the value is 0.0" (it would be hard to tell if there was actual 0 or NaN serialized as 0).

```suggestion
                    return string.Empty;
```

Reply:
Hello, @adamsitnik. Yes,`NaN` occurs in result of doubles zero division here: https://github.com/dotnet/BenchmarkDotNet/blob/master/src/BenchmarkDotNet/Mathematics/Statistics.cs#L61, it calls Perfolizer method: https://github.com/AndreyAkinshin/perfolizer/blob/master/src/Perfolizer/Perfolizer/Mathematics/Common/Moments.cs#L31

Propose to return escaped empty string like `"\"\""`, because we have `StringBuilder`, in resulted Json will be: `""`.